zubbi/views.py

```diff
@@ -86,7 +86,21 @@ class IndexView(ZubbiMethodView):
     template_name = "index.html"
 
     def get(self):
-        context = self.get_context()
+        extra_filter = Q("term", private=False)
+        featured_roles = ["tox", "docker-image", "upload-artifactory"]
```

Comment:
can you move "start" roles to configuration file ?

Reply:
done